mnelab/mainwindow.py

```diff
@@ -360,7 +360,9 @@ def data_changed(self):
                 enabled and self.model.current["ftype"] in ["XDF", "XDFZ", "XDF.GZ"]
             )
             self.actions["convert_od"].setEnabled(
-                len(mne.pick_types(self.model.current["data"].info, fnirs="fnirs_raw"))
+                len(mne.pick_types(
+                    self.model.current["data"].info,fnirs="fnirs_cw_amplitude")
+                )
```

Comment:
Flake didn't like my first push. I tried to copy your style here. I can modify as you want.